### PR TITLE
Version 1.2.0

### PR DESCRIPTION
**Pull request by OnTapKristjan**
Changelog
- Compatibility with Prestashop 1.7.5.0 and 1.7.6.0
- Removed standard (form) integration

### CODE REVIEW

File: views/templates/front/payment.tpl [hunk 1]
```diff
@@ -1,183 +1,153 @@
...
+                    <div class="card-detail left">
+                        <div class='card-detail-label'>Expiry Date</div>
+                        <div class='card-detail-text'>
+                        <span class='left'>{$customer_details-> card->expMonth|escape:'htmlall':'UTF-8'}
+                            / {$customer_details->card->expYear|escape:'htmlall':'UTF-8'}</span>
+
+                            <div id="cc-deletion-container" class="right center">
+                                <div>
+                                    <img id='trash-icon' src="{$module_dir|escape}views/img/trash.png" alt="trash icon"
+                                         title="Delete Credit Card"/>
```
Comment (oconnort) [inline views/templates/front/payment.tpl]: Can you update all instances where it says 'Credit Card' and drop the word credit please?
e.g. Delete Card, Your card has been deleted, New Card etc.

File: views/templates/front/payment.tpl [hunk 2]
```diff
@@ -1,183 +1,153 @@
...
+                            <label class="save" for="saveCustomer" id="updateCustomerLabel">Update your saved card
+                                details?</label>
+                        </div>
+                    {/if}
+
+                </div>
+            </div>
+            <div id="simplify-test-mode-msg" class="test-msg">( TEST PAYMENT )</div>
+            <input type="hidden" name="hostedPayments" value="true"/>
+            <div>
```
Comment (oconnort) [inline views/templates/front/payment.tpl]: Can we remove this empty 'div' tags please.

File: views/templates/front/payment.tpl [hunk 3]
```diff
@@ -1,183 +1,153 @@
...
-    <input type="hidden" name="hostedPayments" value="true"/>
-{/if}
-<div>
-</div>
-</form>
-</div>
-</div>
\ No newline at end of file
+{**
+* Copyright (c) 2017, MasterCard International Incorporated
```
Comment (oconnort) [inline views/templates/front/payment.tpl]: Can you drop uppercase C please: MasterCard -> Mastercard

File: README.md [hunk 4]
```diff
@@ -3,6 +3,12 @@
 
 This plugin adds Simplify Commerce as a payment option on your PrestaShop checkout page.
 
+This module has been tested with following versions of PrestaShop:
```
Comment (oconnort) [inline README.md]: The supporting images need to be updated to reflect the changes to the settings page please.

File: views/templates/front/payment.tpl (hunk 3 above)
Comment (tychobrailleur) [reply views/templates/front/payment.tpl]: and update the year to `(c) 2017–2019`.

**Comment by OnTapKristjan** [comment]
PR moved to https://github.com/simplifycom/simplify-prestashop-module/pull/1 and addressed the feedback items.

**Pull request closed without merging**